crates/javy/src/runtime.rs

```diff
@@ -128,6 +128,10 @@ impl Runtime {
                 unsafe { intrinsic::BignumExt::add_intrinsic(ctx.as_raw()) }
             }
 
+            if intrinsics.contains(JSIntrinsics::STRING_NORMALIZE) {
+                unsafe { intrinsic::StringNormalize::add_intrinsic(ctx.as_raw()) }
```

Comment:
Could you add an entry here? https://github.com/bytecodealliance/javy/blob/main/docs/docs-using-js-api-support.md -- I guess that page needs a general update, so might be worth doing while you're there, if you feel like it.